screeps-game-api/src/inter_shard_memory.rs

```diff
@@ -0,0 +1,35 @@
+//! `inter_shard_memory` provides an interface for communciating between shards.
+//!
+//! Quoted from https://docs.screeps.com/api/#InterShardMemory:
+//!
+//! InterShardMemory object provides an interface for communicating between shards. Your script is executed separatedly
+//! on each shard, and their Memory objects are isolated from each other. In order to pass messages and data between
+//! shards, you need to use InterShardMemory instead.
+//!
+//! Every shard can have its own data string that can be accessed by all other shards. A shard can write only to its
+//! own data, other shards' data is read-only.
+//!
+//! This data has nothing to do with Memory contents, it's a separate data container.
+
+/// Returns the string contents of the current shard's data.
+pub fn get_local() -> String {
+    js_unwrap! {
+        InterShardMemory.getLocal()
```

Comment:
I think it's good since we have js_unwrap!, but it definitely blends in with the other calls to js!. I'll change it to the `()`-braces-oneline-style syntax?

Reply:
Oh, you're right, the macro did everything. Maybe your switch will make this conceptually easier to grok.